src/gui/mzroll/pollyintegration.cpp

```diff
@@ -0,0 +1,263 @@
+#include "pollyintegration.h"
+
+PollyIntegration::PollyIntegration(TableDockWidget* tableDockWidget)
+{
+
+    _tableDockWidget = tableDockWidget;
+}
+
+PollyIntegration::~PollyIntegration()
+{
+    qDebug()<<"exiting PollyIntegration now....";
+}
+
+QByteArray PollyIntegration::run_qt_process(QString command){
+    QProcess process;
+    process.start(command);
+    process.waitForFinished();
+    QByteArray result = process.readAllStandardOutput();
+    QByteArray result2 = process.readAllStandardError();
+    return result;
+}
+
+QByteArray PollyIntegration::run_system_process(QString command){
+    QByteArray ba = command.toLatin1();
+    const char *c_str2 = ba.data();
+    system(c_str2);
+    return "";
+}
+
+
+QString PollyIntegration::get_run_id(QByteArray result){
+    QList<QByteArray> test_list = result.split('\n');
+    int size = test_list.size();
+    QByteArray result2 = test_list[size-2];
+    QJsonDocument doc(QJsonDocument::fromJson(result2));
+    // Get JSON object
+    QJsonObject json = doc.object();
+    QVariantMap json_map = json.toVariantMap();
+    QString run_id =  json_map["id"].toString();
+    qDebug() << run_id;
```

Comment:
There won't be any benefit for printing just a string(run_id) until we print any descriptive word.

Reply:
Removing the Qdebug statement..